### PR TITLE
Fix undefined index/array key warning in `WP_Duotone::colord_parse_hue`

### PR DESCRIPTION
**Pull request by Rahmon**
Trac ticket: https://core.trac.wordpress.org/ticket/59496

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-includes/class-wp-duotone.php [hunk 1]
```diff
@@ -209,10 +209,7 @@ private static function colord_parse_hue( $value, $unit = 'deg' ) {
 			'rad'  => 360 / ( M_PI * 2 ),
 		);
 
-		$factor = $angle_units[ $unit ];
-		if ( ! $factor ) {
-			$factor = 1;
-		}
+		$factor = $angle_units[ $unit ] ?? 1;
```
Comment (mukeshpanchal27) [inline src/wp-includes/class-wp-duotone.php]: The `??` operator usage decision is still pending. Please refer to the discussion here: [link](https://wordpress.slack.com/archives/C02RQBWTW/p1706780040940549?thread_ts=1706779467.750519&cid=C02RQBWTW).

File: tests/phpunit/tests/block-supports/duotone.php [hunk 2]
```diff
@@ -124,4 +124,31 @@ public function data_is_preset() {
 			'invalid'                         => array( 'not a valid attribute', false ),
 		);
 	}
+
+	/**
+	 * @dataProvider data_colord_parse_hue
+	 */
+	public function test_colord_parse_hue( $value, $unit, $expected ) {
```
Comment (mukeshpanchal27) [inline tests/phpunit/tests/block-supports/duotone.php]: Missing `@ticket` annotation.

File: src/wp-includes/class-wp-duotone.php (hunk 1 above)
Comment (Rahmon) [reply src/wp-includes/class-wp-duotone.php]: I thought it was accepted :grimacing:. Updated it.

File: tests/phpunit/tests/block-supports/duotone.php (hunk 2 above)
Comment (Rahmon) [reply tests/phpunit/tests/block-supports/duotone.php]: added :heavy_check_mark:

**Comment by swissspidy** [comment]
Committed in https://core.trac.wordpress.org/changeset/57652

**Comment by rockitround** [comment]
Committed in https://core.trac.wordpress.org/changeset/57656
{{{
#!html
<a href="https://trademinor.org/"></a><a href="https://bitcoinistltd.com/"></a><a href="https://totimescanews.ca"></a><a href="https://hnmag.co.uk/"></a><a href="https://ukhorizon.co.uk"></a><a href="https://britanniabloom.co.uk/"></a><a href="https://britishvista.co.uk/"></a><a href="https://englandecho.co.uk/"></a><a href="https://viralreel.org/"></a><a href="https://fool.social"></a><a href="https://jobsinfoall.com/"></a><a href="http://myblogvista.com/"></a><a href="https://mytimesworld.com"></a><a href="https://networkustad.co.uk"></a><a href="https://itsdailytimes.com/"></a><a href="https://gbcspa.com"></a><a href="https://Socialstarage.com"></a><a href="https://www.digitalseostudio.com"></a><a href="https://googlebusinessprofile.co.nz"></a><a href="https://A1skiphire.ie"></a>
}}}
thank you

**Pull request closed without merging**